cpp/lib/result_set_arrow.cpp

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2021 Snowflake Computing, Inc. All rights reserved.
+ */
+
+#include <string>
+
+#include "memory.h"
+#include "result_set_arrow.h"
```

Comment:
file result_set_arrow.h is not available!!

Reply:
Sorry, it should be available now.